x/programs/rust/wasmlanche-sdk/tests/test-crate/src/lib.rs

```diff
@@ -3,17 +3,16 @@
 use wasmlanche_sdk::{public, Context};
 
 #[public]
-pub fn noop(_: Context) -> i64 {
+pub fn always_true(_: Context) -> i64 {
     true as i64
 }
 
 #[public]
-pub fn return_id(context: Context) -> u32 {
+pub fn combine_last_bit_of_each_id_byte(context: Context) -> u32 {
```

Comment:
naming nit: nicer, perhaps `calculate_id_checksum`?

Reply:
Calling this a checksum is a bit of a stretch haha. But that is more or less what I'm trying to do. This test isn't great though, I'll try and think of a better way to test this.